dsgrid/config/dimensions.py

```diff
@@ -696,6 +696,16 @@ def check_times(
             return ranges
         return _check_time_ranges(ranges, info.data["str_format"], timedelta(days=365))
 
+    @field_validator("measurement_type")
+    @classmethod
+    def check_measurement_type(cls, measurement_type: MeasurementType) -> MeasurementType:
+        # This restriction exists because any other measurement type would require a frequency,
+        # and that isn't part of the model definition.
+        if measurement_type != MeasurementType.TOTAL:
```

Comment:
Isn't this a problem for all time dimensions?

Reply:
Maybe right now, when we only really support energy units, but we need to expand to also support other units, e.g., power, for which other measurement types (instantaneous, mean) would make sense.